test/cppsuite/tests/run.cpp

```diff
@@ -180,8 +183,8 @@ main(int argc, char *argv[])
     std::string cfg, config_filename, current_cfg, current_test_name, home, test_name,
       wt_open_config;
     int64_t error_code = 0;
-    const std::vector<std::string> all_tests = {"reverse_split", "bounded_cursor_perf",
-      "bounded_cursor_prefix_indices", "bounded_cursor_prefix_search_near",
+    const std::vector<std::string> all_tests = {"background_compact", "reverse_split",
```

Comment:
It's not you but can you move `reverse_split` in the right position so it is A - Z ?

Reply:
Fixed https://github.com/wiredtiger/wiredtiger/commit/b21630dc0d16a3e94d7d729cc9d519ab60c84c7e.